client/components/Users/UserFormField.jsx

```diff
@@ -61,6 +61,15 @@ export default class UserFormField extends Component {
         if (validate) additionalOptions.validate = validate;
         return (this.getFieldComponent(field, Select, additionalOptions));
       }
+      case 'InputVirtualizedSelect': {
+        const additionalOptions = {
+            options: field.options,
+            multi: field.multi,
+            displayLabelOnly: field.displayLabelOnly
```

Comment:
There seems to be extra indentation on this object.

Reply:
updated please check